src/classes/DataFactoryTest.cls

```diff
@@ -4,8 +4,8 @@ public class DataFactoryTest {
     @IsTest
     public static void testDefault() {
         AccountDataFactory adf = new AccountDataFactory(Account.SObjectType);
-        Account a = (Account) adf.getDefault();
-        Account b = (Account) adf.getDefault();
+        Account a = (Account) adf.getDefaultInstance();
+        Account b = (Account) adf.getDefaultInstance();
```

Comment:
This variable doesn't seem to have a very insightful name. Please give variables names that suggest their purpose.